index.js

```diff
@@ -153,134 +52,81 @@ function getSha(imageName) {
+function getLatestImageUrl(PROJECTS, projectName, headSha) {
+  const ecrUrl = PROJECTS.filter(project => project["ecrName"] == projectName)[0].ecrUrl
+  return `${ecrUrl}/${projectName}:${shortSha(headSha)}`;
 }
 
-async function buildLogs(projects) {
-  let logs = projects.map(async (project) => {
-    const msgsCommits = await getCommitMessages(project.name, project.oldSha);
-    const strCommits = msgsCommits.join("\n");
-    const projectName = project.name.toUpperCase();
-    return `${projectName}\n\n${strCommits}`;
-  });
-  logs = await Promise.all(logs);
-
-  logs = logs.join("\n\n");
-  if (await isNotLatestManifestsVersion()) {
-    logs = `⚠️ **The production version of manifests is behind the latest staging version. Consider upgrading to the latest version before merging this pull request.** \n\n ${logs}`;
-  }
-
-  if (await isNotLatestTerraformVersion()) {
-    logs = `⚠️ **The production version of the Terraform infrastructure is behind the latest staging version. Consider upgrading to the latest version before merging this pull request.** \n\n ${logs}`;
-  }
-  return logs;
-}
-
-async function getContents(owner, repo, path) {
-  const { data: data } = await octokit.repos.getContents({
-    owner,
-    repo,
-    path,
-  });
-  return data;
-}
-
-async function createPR(
-  projects,
-  issueContent,
-  releaseConfig,
-  newReleaseContentBlob
-) {
-  const branchName = `release-${new Date().getTime()}`;
-  const manifestsSha = await getHeadSha("notification-manifests");
-  const logs = await buildLogs(projects);
-
-  const ref = await octokit.git.createRef({
-    owner: GH_CDS,
-    repo: "notification-manifests",
-    ref: `refs/heads/${branchName}`,
-    sha: manifestsSha,
-  });
-
-  const manifestUpdates = projects
-    .map((project) => {
-      return `${project.name}:${shortSha(project.headSha)}`;
-    })
-    .join(" and ");
-  const update = await octokit.repos.createOrUpdateFile({
-    owner: GH_CDS,
-    repo: "notification-manifests",
-    branch: branchName,
-    sha: releaseConfig.sha,
-    path: "env/production/kustomization.yaml",
-    message: `Updated manifests to ${manifestUpdates}`,
-    content: newReleaseContentBlob,
-  });
-
-  const pr = await octokit.pulls.create({
-    owner: GH_CDS,
-    repo: "notification-manifests",
-    title: `[AUTO-PR] Automatically generated new release ${new Date().toISOString()}`,
-    head: branchName,
-    base: "main",
-    body: issueContent.replace(
-      "> Give details ex. Security patching, content update, more API pods etc",
-      logs
-    ),
-    draft: true,
-  });
-  return Promise.all([ref, update, pr]);
-}
-
-async function hydrateWithSHAs(releaseConfig, projects) {
+async function hydrateWithSHAs() {
   return await Promise.all(
-    releaseConfig.images.map(async (image) => {
-      const matchingProject = projects.find((project) =>
-        image.newName.includes(project.ecrName)
+    PROJECTS.map(async (project) => {
```

Comment:
I prefer if we keep the parameters instead of using globals. It was this way before and I moved it to use parameters so it would be more testable. I know we don't have tests at the moment, but if it keeps growing, it would be good to have and tying global state within functions isn't best.